model/src/test/kotlin/IdentifierTest.kt

```diff
@@ -61,47 +61,49 @@ class IdentifierTest : StringSpec() {
             }
         }
 
-        "Identifier is serialized to String" {
+        "be serialized correctly" {
             val id = Identifier("type", "namespace", "name", "version")
 
             val serializedId = yamlMapper.writeValueAsString(id)
 
             serializedId shouldBe "--- \"type:namespace:name:version\"\n"
         }
 
-        "Identifier can be deserialized from String" {
+        "be deserialized correctly" {
             val serializedId = "--- \"type:namespace:name:version\""
 
             val id = yamlMapper.readValue<Identifier>(serializedId)
 
             id shouldBe Identifier("type", "namespace", "name", "version")
         }
 
-        "Incomplete Identifier can be deserialized from String" {
+        "be deserialized correctly even if incomplete" {
             val serializedId = "--- \"type:namespace:\""
 
             val id = yamlMapper.readValue<Identifier>(serializedId)
 
             id shouldBe Identifier("type", "namespace", "", "")
         }
 
-        "Identifier map key can be deserialized from String" {
+        "be deserialized correctly from a map key" {
             val serializedMap = "---\ntype:namespace:name:version: 1"
 
             val map = yamlMapper.readValue<Map<Identifier, Int>>(serializedMap)
 
             map shouldBe mapOf(Identifier("type", "namespace", "name", "version") to 1)
         }
 
-        "Incomplete Identifier map key can be deserialized from String" {
+        "be deserialized correctly from a map key even if incomplete" {
             val serializedMap = "---\ntype:namespace:: 1"
 
             val map = yamlMapper.readValue<Map<Identifier, Int>>(serializedMap)
 
             map shouldBe mapOf(Identifier("type", "namespace", "", "") to 1)
         }
+    }
 
-        "Checking the vendor works as expected" {
+    "Checking the organization" should {
```

Comment:
We often use the name of the function under test, in this case it would be `"isFromOrg()" should {`.

Reply:
Indeed, we do that sometimes, but I don't like it, as IDE-refactoring does not catch such occurences (well) when renaming functions. Thus I prefer to use nicely human-readable sentences here.